src/widgets/Scrollbar.cpp

```diff
@@ -120,7 +120,7 @@ void Scrollbar::setDesiredValue(qreal value, bool animated)
     value = std::max(this->minimum_,
                      std::min(this->maximum_ - this->largeChange_, value));
 
-    if (std::abs(this->desiredValue_ + this->smoothScrollingOffset_ - value) >
+    if (std::abs(this->currentValue_ + this->smoothScrollingOffset_ - value) >
```

Comment:
Not sure if this is exactly needed but it seems reasonable to do, since the start value is set to `this->currentValue_ + this->smoothScrollingOffset_` and the end value is set to `value` and we want to find out if we need to animate.

If this is wrong then there should be a comment above why we need `this->desiredValue_`.